versionInfo.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "1.0.8.1",
+  "version": "1.0.8.2",
```

Comment:
why there is a difference? i think your branch is from release\1.0.8, did you merge latest changes from release\1.0.8 to your PR branch?

Reply:
This is an error in the diff. Not sure why it's happening. I talked to Damon and he says it sometimes occurs when you merge commits from a branch, then open a PR into that same branch.